lib/Container.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * This file is part of RSS-Bridge, a PHP project capable of generating RSS and
+ * Atom feeds for websites that don't have one.
+ *
+ * For the full license information, please view the UNLICENSE file distributed
+ * with this source code.
+ *
+ * @package Core
+ * @license http://unlicense.org/ UNLICENSE
+ * @link    https://github.com/rss-bridge/rss-bridge
+ */
+
+use Psr\Container\ContainerInterface;
```

Comment:
We do not need the PSR-11 interface for now – it will become important once we want to allow library consumers to use their own DI container.